packages/@aws-cdk/custom-resource-handlers/package.json

```diff
@@ -35,6 +35,9 @@
     "@aws-sdk/client-codepipeline": "3.421.0",
     "@aws-sdk/client-redshift": "3.421.0",
     "@aws-sdk/client-account": "3.421.0",
+    "@aws-sdk/client-eks": "3.421.0",
+    "@aws-sdk/client-sts": "3.421.0",
+    "@aws-sdk/node-http-handler": "^3.370.0",
```

Comment:
Can you confirm this dependency version across the repo and sync if its different

Reply:
Looks like this is the only occurrence: https://github.com/search?q=repo%3Aaws%2Faws-cdk+%22%40aws-sdk%2Fnode-http-handler%22%3A&type=code